src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj

```diff
@@ -16,6 +16,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit" Version="$(MicrosoftCodeAnalysisCSharpCodeFixTestingXUnitVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- Manual exclusions for obsolete interop assemblies -->
```

Comment:
📝 Everything inside an item group marked with this comment is expected to be removed shortly after the initial change. It only has this form because some dependencies are not yet available.